frontend/src/components/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,147 @@
+import Axios from 'axios'
+import React, { useState, useEffect } from 'react'
+import { Redirect } from 'react-router-dom';
+import '../../config';
+import './EditProfile.scss'
+
+export default function EditProfile() {
+    const [loading, setLoading] = useState(true);
+    const [displayName, setDisplayName] = useState("");
+    const [biography, setBiography] = useState("");
+    const [email, setEmail] = useState("");
+    const [birthday, setBirthday] = useState(new Date());
+    const [editStatus, setEditStatus] = useState("profile");
+
+    useEffect(() => {
+        const fetchCurrentUser = () => {
+            Axios.get(global.config.BACKEND_URL + "/account").then(
+                (response) => {
+                    setDisplayName(response.data.displayName);
+                    setBiography(response.data.biography);
+                    setEmail(response.data.email);
+                    setBirthday(response.data.dateOfBirth);
+                    setLoading(false);
+                }).catch(() => {
+                    return <Redirect to="/register"></Redirect>
+                })
+        }
+        fetchCurrentUser();
+    }, [loading])
+
+    useEffect(() => {
```

Comment:
Would it not be possible to add those parameters to the above `useEffect()` and have the same effect?

Reply:
I didn't want it to call the fetch when these parameters changes. I simply want the JSX to re-render.

Reply:
Oh, I see. I didn't know that was a thing. It just felt weird to leave a useEffect emtpy haha